flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
:-(

but when you just repack a binary...

Reply:
Yeah, it's sad. The bass24-linux.zip also comes with an x86 binary, but I haven't figured out how to wire that up in the build system yet. I'm sure it's possible, though.

BASS has no support for Linux on other architectures. In the longer run, I want to get rid of this dependency, and use something open source, but I have been unable to find something that fits my needs so far. And writing a replacement is a bit more work than I currently have the time for...